versionpb/api/version.pb.go

```diff
@@ -1,19 +1,14 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.23.0
-// 	protoc        v3.12.3
+// 	protoc-gen-go v1.34.1
+// 	protoc        (unknown)
```

Comment:
is that expected?

Reply:
> is that expected?

Yeah I think so. As I understand, buf uses its own compiler instead of `protoc` so there won't be any version to report for it.